client/src/components/CreateCampaignStep2.js

```diff
@@ -1,63 +1,248 @@
 /* eslint-disable react/prop-types, no-shadow */ /* - TODO: Fix and remove this line */
 import React from 'react';
 import { connect } from 'react-redux';
-import { Link } from 'react-router';
+import { withRouter, Link } from 'react-router';
+import PropTypes from 'prop-types';
+import {
+  Col,
+  PageHeader,
+  Form,
+  FormGroup,
+  ControlLabel,
+  FormControl,
+  Button
+} from 'react-bootstrap';
 import { updateNewCampaign } from '../redux/actions/newCampaign';
+import { fetchWasteProviders } from '../redux/actions/wasteProvider';
 
-const CreateCampaignStep2 = (props) => {
-  const { updateNewCampaign, router } = props;
+const WASTE_PROVIDER_NOT_SET_ID = 'WASTE_PROVIDER_NOT_SET_ID';
 
-  const setOptionalInfo = async (e) => {
+class CreateCampaignStep2 extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.defaultProvider = {
+      _id: WASTE_PROVIDER_NOT_SET_ID,
+      name: 'Select your provider(Optional)',
+      phone: 'N/A',
+      email: 'N/A'
```

Comment:
Suggest changing N/A to blank if waiting to populate from dropdown.

Reply:
ok